en/editor/project/index.md

```diff
@@ -81,6 +169,66 @@ If you want to replace all the preset options currently used for image asset, yo
 
 ![Layers-camera](./index/layers-camera.png)
 
+## Physics
+
+![Physics](./index/physics-index.png)
+
+This config of physics will effect only when **Physics Module** is enabled in **Engine Module**.
+
+![Physics-In-Engine](./index/physics-in-engine.png)
+
+The physical configuration will be used in the project preview and the code released by the project. The physical effects are reflected in the control of gravity, friction, kinetic energy transfer, and collision detection.
```

Comment:
The physical configuration will be used in the project preview and the code released by the project.

What does this mean? I don't understand "the code released by the project"

Reply:
The physical effects are reflected in the control of gravity, friction, kinetic energy transfer, and collision detection.
-->
The physical effects are reflected in the control of *gravity*, *friction*, *kinetic energy transfer*, and *collision detection*.